internal/pkg/mapper/sharedcode/codes/save.go

```diff
@@ -40,26 +43,33 @@ func (m *mapper) MapBeforeLocalSave(recipe *model.LocalSaveRecipe) error {
 func (w *writer) save() error {
 	// Load content from config row JSON
 	rowContent := w.configRow.Content
-	normalizeContent(rowContent)
 
 	// Load content
 	raw, found := rowContent.Get(model.ShareCodeContentKey)
 	if !found {
 		return fmt.Errorf(`key "%s" not found in %s`, model.ShareCodeContentKey, w.configRow.Desc())
 	}
 
-	// Content must be string
-	codeContent, ok := raw.(string)
-	if !ok {
-		return fmt.Errorf(`key "%s" must be string in %s`, model.ShareCodeContentKey, w.configRow.Desc())
-	}
-
 	// Get target component of the shared code -> needed for file extension
 	targetComponentId, err := w.GetTargetComponentId(w.config)
 	if err != nil {
 		return err
 	}
 
+	// Content must be string or array
+	var codeContent string
+	if codeContentStr, ok := raw.(string); ok {
```

Comment:
One more fix:

Shared code bol povodne `string`, pod klucom `code_content`.
To ale neumoznovalo aby v shared code bolo viac SQL queries (kedze sa delia na `string[]` na strane UI).

Preto je po novom `code_content` `[]string`, PS to upravil.
V legacy configs moze byt este stale `string`.
A preto pridavam podporu pre `[]string`. 
Uklada sa vzdy `[]string` a nacitanie podporuje aj `string`, aj `[]string`.

![image](https://user-images.githubusercontent.com/19371734/141322117-87038e30-164a-4092-8a96-5d9b2afece67.png)